examples/message/labview/LabVIEW.cs

```diff
@@ -0,0 +1,183 @@
+﻿using System;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using NationalInstruments.SystemLink.Clients.Message;
+
+namespace NationalInstruments.SystemLink.Clients.Examples.Message
+{
+    /// <summary>
+    /// Example for the SystemLink Message Client API that communicates with
+    /// the Host.vi from the Async Messaging LabVIEW example.
+    /// </summary>
+    class LabVIEW
+    {
+        /// <summary>
+        /// The delay between sending periodic messages to the host topic.
+        /// </summary>
+        static readonly TimeSpan MessageDelay = TimeSpan.FromSeconds(1);
+
+        /// <summary>
+        /// The message topic for sending messages to the Host VI.
+        /// </summary>
+        const string HostTopic = "Example to Host";
+
+        static void Main(string[] args)
+        {
+            /*
+             * See the configuration example for how a typical application
+             * might obtain a configuration.
+             */
+            var configuration = ExampleConfiguration.Obtain(args);
```

Comment:
It is mentioned below that this doesn't support Cloud, and we specifically instruct users to avoid passing in the `--cloud` argument. What happens if they do so anyway? Should we check for this here and surface a message that this is unsupported?

Reply:
If the LabVIEW example and the C# example are configure for different servers (or cloud), they won't be able to communicate with each other. I think it could make sense to prevent the use of --cloud in this example, and we can investigate making the LV example support cloud in a future release.